components/iotserver-mgt/org.wso2.carbon.device.mgt.iot.common/src/main/java/org/wso2/carbon/device/mgt/iot/common/transport/mqtt/MqttConfig.java

```diff
@@ -0,0 +1,55 @@
+package org.wso2.carbon.device.mgt.iot.common.transport.mqtt;
+
+import org.wso2.carbon.device.mgt.iot.common.config.server.DeviceCloudConfigManager;
+import org.wso2.carbon.device.mgt.iot.common.config.server.datasource.ControlQueue;
+
```

Comment:
MQTT transport has been moved to input & output adapters in [cdmf-device-mgt-plugins](https://github.com/wso2/carbon-device-mgt-plugins/tree/master/components/iot-plugins/iot-base-plugin/org.wso2.carbon.device.mgt.iot.input.adapter.mqtt/src/main/java/org/wso2/carbon/device/mgt/iot/input/adapter/mqtt/util). This class does not exist anymore.